README.md

```diff
@@ -8,11 +8,11 @@ MediaWiki Page: https://www.mediawiki.org/wiki/Extension:PHPBB_Auth
 REQUIREMENTS
 =================
 
-* PHP 7.3 or later
+* PHP 7.3 (not 7.4)
```

Comment:
This is incorrect. We are using PHP 7.4.3 at [Distributed Proofreaders](https://www.pgdp.net/c) and the plugin works fine there. I will look into the error you are seeing in https://github.com/Digitalroot-Technologies/MediaWiki_PHPBB_Auth/issues/58.